src/Microsoft.AspNet.Security.OpenIDConnect/OpenidConnectAuthenticationHandler.cs

```diff
@@ -0,0 +1,580 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.
```

Comment:
Like `OpenIdConnectAuthenticationHandler` in Katana 3, this file has a name whose case doesn't correspond to the class name: `OpenidConnectAuthenticationHandler` -> `OpenIdConnectAuthenticationHandler`.